builder/googlecompute/config_test.go

```diff
@@ -685,17 +684,45 @@ func TestConfigExtraBlockDevice_create_image_multiple(t *testing.T) {
 	}
 }
 
+func TestConfigPrepareImageArchitecture(t *testing.T) {
+	cases := []struct {
+		Architecture string
+		Valid        bool
```

Comment:
Super micro nit: I would name this `ExpectValid` just to be clear that this is the test's expectation and not a parameter.

Reply:
This has been renamed to the suggested value.